api/agent/metricsadder/client_test.go

```diff
@@ -145,12 +145,4 @@ func (s *metricsAdderIntegrationSuite) TestAddMetricBatches(c *gc.C) {
 	result, ok := results[batches[0].Batch.UUID]
 	c.Assert(ok, jc.IsTrue)
 	c.Assert(result, gc.IsNil)
-
```

Comment:
We can delete this whole file. It uses old testing methodology and we don't want to retain it just to test a no op stub. The whole metricsadder api package can be deleted.